modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib

```diff
@@ -683,7 +683,7 @@ @article{Mikelic2019
 }
 
 @phdthesis{Bolourchi1979,
-Author = {Bolourchi, Sa{\''i}d},
+Author = {Bolourchi, Sa{\"\i}d},
```

Comment:
Would `{\"i}` work? Instead of `{\''\i}`

Reply:
no, you would get 3 dots. `\i` removes the dot from `i`.

Reply:
hold on, maybe it would... just trying it... this used to be a thing...

Reply:
ok, it is no longer a thing. `\"i` works too.